### PR TITLE
add import proportions button

### PR DESCRIPTION
**Pull request by ImUrX**

### CODE REVIEW

**Review by Louka3000 — CHANGES_REQUESTED** [review]
Fails on import

**Review by Louka3000 — APPROVED** [review]

File: gui/src/components/onboarding/pages/body-proportions/ProportionsChoose.tsx [hunk 1]
```diff
@@ -214,9 +251,81 @@ export function ProportionsChoose() {
...
+                    );
+                    return setImportState(ImportStatus.FAILED);
+                  }
+                }
+
+                parseConfigImport(config).forEach((req) =>
+                  sendRPCPacket(RpcMessage.ChangeSkeletonConfigRequest, req)
+                );
+                setImportState(ImportStatus.SUCCESS);
+              }}
```
Comment (loucass003) [inline gui/src/components/onboarding/pages/body-proportions/ProportionsChoose.tsx:296]: Put this in a function please

**Review by ButterscotchV — COMMENTED** [review]
I've looked at it and it looks fine to me, but I can't really say whether it's good or not

**Review by loucass003 — APPROVED** [review]
LGTM

**Pull request merged**